src/table/components/TableWrapper.jsx

```diff
@@ -152,13 +154,23 @@ export default function TableWrapper(props) {
         data-testid="table-container"
       >
         <Table stickyHeader aria-label={tableAriaLabel}>
-          <TableHeadWrapper {...props} />
+          <TableHead>
+            <TableHeadWrapper
+              {...props}
+              setFocusedCellCoord={setFocusedCellCoord}
+              focusedCellCoord={focusedCellCoord}
+            />
+            {totalsPosition === 'top' && <TableTotals {...props} setFocusedCellCoord={setFocusedCellCoord} />}
+          </TableHead>
           <TableBodyWrapper
             {...props}
             announce={announce}
             setShouldRefocus={setShouldRefocus}
             tableWrapperRef={tableWrapperRef}
           />
+          <TableFooter>
+            {totalsPosition === 'bottom' && <TableTotals {...props} setFocusedCellCoord={setFocusedCellCoord} />}
+          </TableFooter>
```

Comment:
We don't want the footer at all if there is no totals here
```suggestion
          {totalsPosition === 'bottom' && (<TableFooter>
            <TableTotals {...props} setFocusedCellCoord={setFocusedCellCoord} />
          </TableFooter>)}
```